mariadb/templates/mariadb-deployment.yaml

```diff
@@ -1,40 +1,36 @@
-apiVersion: v1
-kind: ReplicationController
+apiVersion: extensions/v1beta1
+kind: Deployment
 metadata:
-  name: {{ printf "%s-%s" .Release.Name "mariadb" | trunc 24 }}
+  name: {{ template "fullname" . }}
   labels:
```

Comment:
Do we want to add a `chart` label (`chart: "{{.Chart.Name}}-{{.Chart.Version}}"`) as used in https://github.com/bitnami/charts/pull/61/files#diff-b2f59af60ea292fbe4ba76016ce2a8cfR13? Since we already have `provider` and `version` labels wouldn't it be redundant?